### PR TITLE
Infinite scroll

### PR DESCRIPTION
**Pull request by ak29-dev**
**Description of PR**
Feature implementation of infinite scroll in browser and mobile versions of the essentials page. I've removed the pagination section
**Type of PR**

- [ ] Bugfix
- [X] New feature

**Relevant Issues**  
Fixes #...

**Checklist**

- [X] Compiles and passes lint tests
- [X] Properly formatted
- [X] Tested on desktop
- [X] Tested on phone

**Screenshots**
image

image

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -285,6 +285,18 @@ h6 {
...
+  .filtersection{
+    position: sticky;
+    top: 0;
+    z-index: 100;
+    background: white;
+    width: 100%;
+  }
+
+  .resourcesaccordion{
+    width: 336px; //find a better solution to this
```
Comment (sama007) [inline src/App.scss]: It has to be 100%. Check please.

File: src/components/mapexplorer.js [hunk 2]
```diff
@@ -393,7 +393,11 @@ export default function ({
   const {name, lastupdatedtime} = currentHoveredRegion;
 
   useEffect(() => {
-    setTestObj(stateTestData.find((obj) => obj.state === panelRegion.name));
+    setTestObj(
```
Comment (sama007) [inline src/components/mapexplorer.js:396]: Why changes in this file?? Is this file relevant to this tab?

**Review by sama007 — CHANGES_REQUESTED** [review]
Please check the added comments

**Review by sama007 — APPROVED** [review]
Resolve conflicts please

**Comment by nikita-11** [comment]
resolved

**Review by nikita-11 — APPROVED** [review]
